tests/console/yast2_nfs_server.pm

```diff
@@ -83,23 +87,20 @@ sub run {
     clear_console;
 
     validate_script_output "exportfs", sub { m,/srv/nfs, };
-    if (get_var('NFSSERVER')) {
-        assert_script_run 'mount 10.0.2.1:/ /mnt';
-    }
-    else {
-        assert_script_run 'mount 10.0.2.15:/ /mnt';
-    }
-
-    # Timeout of 95 seconds to account for the NFS server grace period
-    validate_script_output "cat /mnt/file", sub { m,mounted, }, 95;
-    assert_script_run 'umount /mnt';
 
     if (get_var('NFSSERVER')) {
         # Server is up and running, client can use it now!
         mutex_create('nfs_ready');
         # Wait for the children (nfs clients) to finish
         wait_for_children;
     }
+    else {
+        assert_script_run 'mount 10.0.2.15:/ /mnt';
+
+        # Timeout of 95 seconds to account for the NFS server grace period
+        validate_script_output "cat /mnt/file.txt", sub { m,success, }, 95;
```

Comment:
Don't move this section. The mutex is intentionally after verifying that the server is up.

Reply:
Now it's back.